webtau-core/src/test/groovy/com/twosigma/webtau/expectation/equality/handlers/HandlerMessagesTest.groovy

```diff
@@ -232,6 +232,133 @@ class HandlerMessagesTest {
         } should throwException(AssertionError, ~/expected: not 1/)
     }
 
+    // ArrayAndIterableCompareToHandler messages
```

Comment:
I can't remember the purpose of this super test class now :( Do we need to expand it with every new handler? Should we move code-throw cases to the handler test itself?

Reply:
I don't remember the purpose either but as it seemed to have some tests for virtually all handlers I figured I should add some too.

I agree it would make more sense for these to live with the relevant handler tests but perhaps that can be a cleanup effort for a separate PR.

Reply:
I feel like it had handlers, to make sure they are actually wired, but I started to do it in each of the handlers themselves. Feel free to move the one you added to its own test.